lms/static/scripts/frontend_apps/components/LMSPagePicker.tsx

```diff
@@ -0,0 +1,300 @@
+import {
```

Comment:
Aside from the difference in which API calls are made and the terminology presented to the user, are there any other differences between files and pages? If not, this component and its tests are pretty complex and I'd probably lean towards using a single implementation until we have a clear reason to treat them differently.